### PR TITLE
Changing the events.peek() function to show sky image in RA/DEC and event rate

### PR DESCRIPTION
**Pull request by facero**
changing peek() to show image in RA/DEC and event rate vs time. 
Fixed the double plot issue in jupyter for the peek function only. Double plot is due to return fig in plot routine.

### CODE REVIEW

**Comment by cdeil** [comment]
Aha, here it is

:-)

You made the pull request against your own fork at https://github.com/facero/gammapy
Can you please make a pull request against the main Gammapy repo at https://github.com/gammapy/gammapy instead?
You can re-use the same branch, and just make a new pull request and close this one.

---

Some comments:

* Can you remove the `plt.show` and `return fig` completely? You currently have it commented out, but I think we know what we want now and can just clean that out.
* Rename `plot_imageRADEC` to `plot_image_radec` or `plot_skyimage` (basically, PEP8 style guide wins) :-)


For the sky image, maybe calling [SkyImage.empty](http://docs.gammapy.org/en/latest/api/gammapy.image.SkyImage.html#gammapy.image.SkyImage.empty) and to put a local TAN projection, and [SkyImage.fill_events](http://docs.gammapy.org/en/latest/api/gammapy.image.SkyImage.html#gammapy.image.SkyImage.fill_events) and `SkyImage.plot` would be better?

So it could work like that:
```
events = EventList.read('...')
events.counts_image(<options?>).plot(<options>)
```
where the `counts_image` method would call something like
```python
def counts_image(<options?>):
    image = events._empty_image(<options?>)
    image.fill(self)
    return image
```

So basically, it would be a convenience, re-exposing the existing methods from `SkyCoord` on `EventList`, so that if you have an eventlist in hand, it's quick to get a counts image and plot.

I haven't thought much whether this is a good idea, or if what you do here (re-implementing histogramming, not really having a sky image with a WCS) is better for some reason.

@facero - Thoughts?
